src/api/service/user_service.py

```diff
@@ -0,0 +1,8 @@
+from repository.user_repository import UserRepository
+
+class UserService:
+    def __init__(self):
+        self.user_repository = UserRepository()
+
+    def get_user(self, user_id: int):
+        return self.user_repository.get_user_by_id(user_id)
```

Comment:
I think we need a validation to check if the db call has data.

That is, we must check if the repository call returns any data.
If no data exist, then it means given user_id does not exist. 
In that case service should throw some kind of error, this should be handled in controller and throw 404 NOT_FOUND http response with error message.

This needs design pattern, I will work on this. For now, let it be like this.

Reply:
ok, I will leave as it is